modules/public/logic.py

```diff
@@ -162,6 +213,12 @@ def convert_limit_offset_to_size_page(url, link):
 
 
 def format_paragraphs(unformatted_description):
+    """Format paragraphs to displayable paragraph
```

Comment:
split_description_into_paragraphs()
"""
Split a long description into a set of paragraphs. We assume each paragraph is separated by 2 or more line-breaks in the description.